src/main/java/uk/gov/hmcts/probate/service/notification/GrantOfRepresentationPersonalisationService.java

```
@@ -187,22 +174,13 @@ private StringBuilder getExelaBuiltData(List<ReturnedCaseDetails> cases) {
         return data;
     }
 
-    private StringBuilder getDraftCasesBuiltData(List<ReturnedCaseDetails> cases) {
-        StringBuilder data = new StringBuilder();
-
-        for (ReturnedCaseDetails currentCase : cases) {
-            getCaseData(data, currentCase.getId(), currentCase.getData().getDeceasedForenames(),
-                    currentCase.getData().getDeceasedSurname());
-        }
-        return data;
-    }
-
-    private StringBuilder getCaveatDraftCasesBuiltData(List<ReturnedCaveatDetails> cases) {
+    private StringBuilder getDraftCasesBuiltData(List<uk.gov.hmcts.reform.ccd.client.model.CaseDetails> cases) {
         StringBuilder data = new StringBuilder();
 
-        for (ReturnedCaveatDetails currentCase : cases) {
-            getCaseData(data, currentCase.getId(), currentCase.getData().getDeceasedForenames(),
-                    currentCase.getData().getDeceasedSurname());
+        for (uk.gov.hmcts.reform.ccd.client.model.CaseDetails currentCase : cases) {
+            getCaseData(data, currentCase.getId(),
```

it's not changed here, but this method doesn't log the fact that it's catching `Exception`, it just silently swallows it, terminates the current record and then continues - this feels like a good way to get junk data and have no record of the fact later